src/components/apps/admin/details/AdminAppDetails.js

```diff
@@ -0,0 +1,324 @@
+}
+
+function AdminAppDetailsForm(props) {
+    const {
+        parentId,
+        restrictedChars,
+        restrictedStartingChars,
+        documentationTemplateUrl,
+        handleClose,
+    } = props;
+
+    const { t } = useTranslation("apps");
+    const { t: i18nCommon } = useTranslation("common");
+    return (
+        <Form>
+            <Field
+                name={"name"}
+                label={t("name")}
+                id={build(parentId, ids.ADMIN_DETAILS.NAME)}
+                validate={(value) =>
+                    validateAppName(
+                        restrictedStartingChars,
+                        restrictedChars,
+                        value
+                    )
+                }
+                component={FormTextField}
+            />
+            <Field
+                name={"description"}
+                label={t("descriptionLabel")}
+                id={build(parentId, ids.ADMIN_DETAILS.DESCRIPTION)}
+                component={FormMultilineTextField}
+            />
+            <Field
+                name={"integrator_name"}
+                label={t("integratorName")}
+                id={build(parentId, ids.ADMIN_DETAILS.INTEGRATOR)}
+                component={FormTextField}
+            />
+            <Field
+                name={"integrator_email"}
+                label={t("integratorEmail")}
+                id={build(parentId, ids.ADMIN_DETAILS.INTEGRATOR_EMAIL)}
+                component={FormTextField}
+            />
+            <Field
+                name={"extra.htcondor.extra_requirements"}
+                label={t("htcondorExtraRequirements")}
+                id={build(parentId, ids.ADMIN_DETAILS.HTCONDOR_EXTRA_REQS)}
+                component={FormTextField}
+            />
+            <Field
+                name={"deleted"}
+                label={t("deleted")}
+                id={build(parentId, ids.ADMIN_DETAILS.DELETED)}
+                component={FormCheckbox}
+            />
+            <Field
+                name={"disabled"}
+                label={t("disabled")}
+                id={build(parentId, ids.ADMIN_DETAILS.DISABLED)}
+                component={FormCheckbox}
+            />
+            <Field
+                name={"beta"}
+                label={t("beta")}
+                id={build(parentId, ids.ADMIN_DETAILS.BETA)}
+                component={FormCheckbox}
+            />
+            <Paper elevation={1}>
+                {t("documentationInstructions")}
+                <Link
+                    onClick={() =>
+                        window.open(getHost() + "/" + documentationTemplateUrl)
+                    }
+                >
+                    {t("documentationTemplate")}
+                </Link>
+            </Paper>
+            <Field
+                name={"documentation.documentation"}
+                label={t("appDocumentation")}
+                id={build(parentId, ids.ADMIN_DETAILS.DOCUMENTATION)}
+                t
```

Comment:
`t`ypo?

Reply:
Its funny 😂  and frustrating 😩 that how I can get away with these issue without encountering errors when testing.